cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,82 @@
 /// <reference types='cypress' />
+const { generateUser}  = require('../support/commands');
 
 describe('Student Registration page', () => {
   before(() => {
-    
+    cy.visit('https://demoqa.com/automation-practice-form')
   });
 
-  it('', () => {
+  it('User is able to fill a registration form', () => {
+    const{firstName, lastName, email, phone, gender, subjects, hobbies, address} = generateUser()
```

Comment:
```suggestion
    const{firstName, lastName, email, phone, gender, subjects, hobbies, address} = generateUser();
```
check your code if you missed the `;`